src/Fixer/Whitespace/IndentationTypeFixer.php

```diff
@@ -64,7 +64,7 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
     public function getDefinition()
     {
         return new FixerDefinition(
-            'Code MUST use an indent of 4 spaces, and MUST NOT use tabs for indenting.',
+            'Code must use the indent configured (default PSR2 4 spaces).',
```

Comment:
we used uppercase to follow RFC 2119.
if we want to drop it, i would rather do it globally